src/Markdown.MAML/Transformer/ModelTransformerBase.cs

```diff
@@ -478,5 +495,13 @@ protected string GetTextFromParagraphNode(ParagraphNode node)
         /// <param name="command"></param>
         /// <returns>true if Section was found</returns>
         protected abstract bool SectionDispatch(MamlCommand command);
+
+        protected void Report(string warning)
+        {
+            if (_warningCallback != null)
+            {
+                _warningCallback.Invoke("Error encountered: " + warning);
```

Comment:
I'd actually suggest to replace the string to something like `[ModelTransformer]`